packages/paste-core/utilities/flex/src/index.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box} from '@twilio-paste/box';
+import {FlexboxProps} from '@twilio-paste/types';
+
+export type displayOptions = 'flex' | 'inline-flex';
+export type vAlignOptions = 'top' | 'center' | 'bottom' | 'stretch';
+export type hAlignOptions = 'left' | 'center' | 'right' | 'around' | 'between';
+
+export interface FlexProps {
+  display?: displayOptions;
+  vertical?: boolean;
+  vAlignContent?: vAlignOptions;
+  hAlignContent?: hAlignOptions;
+  grow?: boolean | number;
+  shrink?: boolean | number;
+  basis?: string | number;
+  wrap?: boolean;
+}
+
+function getGrow({grow}: FlexProps): number {
+  if (typeof grow === 'number') {
+    return grow;
+  }
+
+  if (grow) {
+    return 1;
+  }
+
+  return 0; // default
+}
+
+function getShrink({shrink, basis}: FlexProps): number {
+  if (typeof shrink === 'number') {
+    return shrink;
+  }
+
+  if (shrink) {
+    return 1;
+  }
+
+  if (shrink === false) {
+    return 0;
+  }
+
+  if (basis && basis !== 'auto') {
+    return 0;
+  }
+
+  return 1; // default
+}
+
+function getBasis({basis}: FlexProps): string {
+  if (basis) {
+    const suffix = typeof basis === 'number' || String(parseInt(basis as string, 10)) === basis ? 'px' : '';
+    return basis + suffix;
+  }
+
+  return 'auto'; // default
+}
+
+function alignPropToFlex(align: FlexProps['vAlignContent'] | FlexProps['hAlignContent']): {} {
+  switch (align) {
```

Comment:
then here check is `Array.isArray(align)` to see if you have to loop the responsive values and generate an array return value